aws/resource_aws_ecs_service.go

```diff
@@ -35,6 +35,31 @@ func resourceAwsEcsService() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"capacity_provider_strategy": {
```

Comment:
This new attribute is missing documentation in `website/docs/r/ecs_service.html.markdown`